hazelcast/test/src/HazelcastTests1.cpp

```diff
@@ -864,19 +1155,12 @@ namespace hazelcast {
                 client_config config;
                 config.get_connection_strategy_config().get_retry_config().set_cluster_connect_timeout(
                         std::chrono::seconds(2)).set_initial_backoff_duration(std::chrono::milliseconds(100));
-                config.set_cluster_name(get_ssl_cluster_name()).get_network_config().add_address(
-                        address("8.8.8.8", 5701)).get_ssl_config().set_enabled(true).add_verify_file(
-                        get_ca_file_path());
-                ASSERT_THROW(hazelcast_client client{new_client(std::move(config)).get()},
-                             exception::illegal_state);
-            }
+                boost::asio::ssl::context ctx(boost::asio::ssl::context::method::tlsv12_client);
+                ctx.set_default_verify_paths();
+                ctx.load_verify_file(get_ca_file_path());
```

Comment:
I believe these are not relevant to this test. They are redundant without setting the verify mode anyways.

```suggestion
```

Reply:
removed.